helm/oauth2-proxy/README.md

```diff
@@ -211,3 +212,27 @@ extraEnv:
   - name: TEST_ENV_VAR_2
     value: '{{ .Values.tplValue }}'
 ```
+
+## htpasswd - Manage credentials for basic authentication
+
+Oauth2-proxy additionally let's you authenticate against a htpasswd file.
+There are two options to provide basic authentication credentials:
+
+1. Generate a htpasswd string with `htpasswd -B` (bcrypt) or `htpasswd -s` (SHA1) and provide it as a list of strings:
```

Comment:
We are deprecating SHA1 so I'd recommend removing this from the instruction, all passwords should be bcrypt'ed

Reply:
Ok, will remove this